shacl/register.ttl

```diff
@@ -167,8 +167,13 @@ reg:DistributionContentUrlProperty a sh:PropertyShape ;
     rdfs:comment "De URL waar de distributie rechtstreeks toegankelijk is."@nl, "The URL where the distribution can be directly accessed."@en ;
     sh:minCount 1 ;
     sh:maxCount 1 ;
+    sh:nodeKind sh:IRI ;
+    sh:pattern "^https?://" ;
+    sh:severity sh:Warning ;
```

Comment:
Changing the severity to warning makes this accept distributions without a content URL, which was required before. So please create a separate constraint with severity warning instead.